### PR TITLE
Soundscape testing code

### LINKED ISSUE #39: Update inference notebook
The inference notebook reflects the past BirdCLEF work when it needed to be run on Kaggle. It needs to be updated to reflect the structural changes we've made and replace the black box .csv output to get useful information. Visuals like a confusion matrix and statistics like IoU could also be useful.

### PR DESCRIPTION
**Pull request by benjamin-cates**

### CODE REVIEW

File: documentation/default_config.yml
Comment (marco-bazzani) [inline documentation/default_config.yml:1]: Add to docs!

File: pyha_analyzer/utils.py
Comment (marco-bazzani) [inline pyha_analyzer/utils.py:1]: What a descriptive docstring

**Review by marco-bazzani — CHANGES_REQUESTED** [review]
See comments

File: documentation/default_config.yml
Comment (benjamin-cates) [reply documentation/default_config.yml:1]: Done!

**Review by marco-bazzani — DISMISSED** [review]

**Review by Sean1572 — DISMISSED** [review]
Approved

**Review by marco-bazzani — APPROVED** [review]

**Pull request merged**